### ISSUE TITLE
This is actually working by design. With 4.7.0 release we decided to finally drop our [good old Paste dialog](https://i.imgur.com/89CSoQF.png) to simplify pasting process as much as possible.

### ISSUE THREAD

**Issue by yogesh-mali44** [ckeditor/ckeditor5-angular#247]
This is actually working by design. With 4.7.0 release we decided to finally drop our [good old Paste dialog](https://i.imgur.com/89CSoQF.png) to simplify pasting process as much as possible.

Ideally we'd like the browser to _just paste_ and that's it - however, due the security reasons, modern browsers do not expose any means to paste other than using native ways, so:
* Platform specific hotkey <kbd>ctrl/cmd</kbd> + <kbd>v</kbd>
* Native context menu

Will trigger paste, but the website can not initiate it on it's own.

The exception here would be Internet Explorer family, as it supports triggering paste command which we use.

So in short this is as close as it gets, and we're unable to push it further. As soon as browser provides APIs for us to trigger the paste, we're happy to utilize that.

_Originally posted by @mlewand in https://github.com/ckeditor/ckeditor4/issues/469#issuecomment-306279192_

**Comment by yogesh-mali44** [comment]
Is this known issue with so fix as of now and is it applicable to all new ckeditor versions including ckeditor5-angular and ckeditor4-angular

**Comment by ma2ciek** [comment]
Hey @yogesh-mali44.

This is an issue tracker for `ckeditor5-angular` related issues. And the issue mentioned by you touches the whole CKEditor 5 framework, so please try to find it on [https://github.com/ckeditor/ckeditor5](https://github.com/ckeditor/ckeditor5/issues) or report a new one there if it's missing, so the issue can be easily found by others and the problem will be discussed in the correct place.

**Comment by yogesh-mali44** [comment]
Don't think this is a new issue, its a known issue on desktops for ck-editor but I did not find its mention anywhere for angular4-ckeditor or angular5-ckeditor. Is there a fix in pipeline for this or is there a workaround?
